app/controllers/buses_controller.rb

```diff
@@ -0,0 +1,22 @@
+class BusesController < ApplicationController
+  def create
+    Bus.create bus_params
```

Comment:
`create!` has been the idiom for this app so that if something goes wrong we'll know about it.